src/Components/UserLogin.jsx

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import { Stack, HStack, VStack, Box, Text } from "@chakra-ui/react";
+import { Image } from "@chakra-ui/react";
+import {
+  FormControl,
+  FormLabel,
+  FormHelperText,
+  Input,
+  Flex,
+  InputGroup,
+  InputRightElement,
+  Button,
+} from "@chakra-ui/react";
+import { useNavigate } from "react-router-dom";
+
+function UserLogin() {
+  const [show, setShow] = React.useState(false);
+  const handleClick = () => setShow(!show);
+  const navigate = useNavigate()
+
+  const createAccount = () =>{
+    navigate("/UserSignup")
+  }
+
+  
+
+  return (
+    <Box
+      display={"flex"}
```

Comment:
General rule

if the values are static then please avoid wrapping them in culy braces
```suggestion
      display="flex"
```

also why not just use `Flex` component in the above case

Reply:
done